src/fides/api/models/privacy_center_config.py

```diff
@@ -0,0 +1,106 @@
+from sqlalchemy import Column
+from sqlalchemy.dialects.postgresql import JSONB
+from sqlalchemy.ext.declarative import declared_attr
+from sqlalchemy.ext.mutable import MutableDict
+
+from fides.api.db.base_class import Base
+
+default_privacy_center_config = {
+    "title": "Take control of your data",
+    "description": "When you use our services, you’re trusting us with your information. We understand this is a big responsibility and work hard to protect your information and put you in control.",
+    "description_subtext": [],
+    "addendum": [],
+    "logo_path": "/logo.svg",
+    "logo_url": "https://fid.es",
+    "privacy_policy_url": "https://fid.es/privacy",
+    "privacy_policy_url_text": "Privacy Policy",
+    "favicon_path": "/favicon.ico",
+    "actions": [
+        {
+            "policy_key": "default_access_policy",
+            "icon_path": "/download.svg",
+            "title": "Access your data",
+            "description": "We will provide you a report of all your personal data.",
+            "description_subtext": [],
+            "identity_inputs": {"email": "required"},
+            "confirmButtonText": "Continue",
+            "cancelButtonText": "Cancel",
+        },
+        {
+            "policy_key": "default_erasure_policy",
+            "icon_path": "/delete.svg",
+            "title": "Erase your data",
+            "description": "We will erase all of your personal data. This action cannot be undone.",
+            "description_subtext": [],
+            "identity_inputs": {"email": "required"},
+            "confirmButtonText": "Continue",
+            "cancelButtonText": "Cancel",
+        },
+    ],
+    "includeConsent": True,
+    "consent": {
+        "button": {
+            "description": "Manage your consent preferences, including the option to select 'Do Not Sell or Share My Personal Information'.",
+            "description_subtext": [
+                "In order to opt-out of certain consent preferences, we may need to identify your account via your email address. This is optional."
+            ],
+            "icon_path": "/consent.svg",
+            "identity_inputs": {"email": "optional"},
+            "title": "Manage your consent",
+            "modalTitle": "Manage your consent",
+            "confirmButtonText": "Continue",
+            "cancelButtonText": "Cancel",
+        },
```

Comment:
Why do we add all these sections in right now when we can't use them or save them?